Update Notes/2.1.md

```diff
@@ -107,6 +111,5 @@
 - Added 4 new textures for the on/off switches in /neu
 - Code Cleanup - IRONM00N
 - Renamed Keebler408 to CraftyOldMiner
-- Added JUnit unit tests for metal detector and wishing compass solver
```

Comment:
Please restore this line.

Reply:
I dont think anyone is gonna understand this.

Reply:
It's like saying "Code Cleanup", but instead it's "Add Tests". That said, I commented on this deletion since it's unrelated to the PR.